### PR TITLE
Add basic test for revealing activity details section

### PR DESCRIPTION
**Pull request by blanchardjeremy**

### CODE REVIEW

File: src/pages/__tests__/Calculator.test.tsx [hunk 1]
```diff
@@ -10,4 +10,21 @@ describe('calculator page', () => {
 
     expect(getByText(/constructed a calculator that lets/i)).toBeInTheDocument()
   })
+
+  it('activity details section reveals after location is selected', () => {
+    const { getByText, queryByText, getByRole } = render(<Calculator />, {
+      wrapper: AllProviders,
+    })
+
+    expect(getByText(/constructed a calculator that lets/i)).toBeInTheDocument()
```
Comment (dominic) [inline src/pages/__tests__/Calculator.test.tsx:19]: This is already tested in the previous test; no need to repeat.

File: src/pages/__tests__/Calculator.test.tsx [hunk 2]
```diff
@@ -10,4 +10,21 @@ describe('calculator page', () => {
...
+    const interactionTypeLabelQuery = /Is this a single activity or an ongoing relationship/i
+    expect(queryByText(interactionTypeLabelQuery)).not.toBeInTheDocument()
+    const topLocationBox = getByRole('combobox', { name: /Select location/i })
+    fireEvent.change(topLocationBox, {
+      target: { value: 'US_06' }, // California
+    })
+    const californiaOption = getByText(/California/i) as HTMLOptionElement
```
Comment (dominic) [inline src/pages/__tests__/Calculator.test.tsx:26]: Can we use this `californiaOption` value to get the select target in the statement above?

File: src/pages/__tests__/Calculator.test.tsx (hunk 1 above)
Comment (blanchardjeremy) [reply src/pages/__tests__/Calculator.test.tsx:19]: Oops

File: src/pages/__tests__/Calculator.test.tsx (hunk 2 above)
Comment (blanchardjeremy) [reply src/pages/__tests__/Calculator.test.tsx]: Oh smart move. Yeah that works. Done.

File: src/pages/__tests__/Calculator.test.tsx (hunk 1 above)
Comment (blanchardjeremy) [reply src/pages/__tests__/Calculator.test.tsx]: Fixed.

**Pull request merged**